articles/connections/enterprise/adfs.md

```diff
@@ -22,45 +22,41 @@ useCase:
     - customize-connections
     - add-idp
 ---
-# Connect your app to ADFS
+# Connect Your Application to Microsoft ADFS
 
-First, provide this information to your ADFS administrator:
+To connect your application to Microsoft ADFS, you will need to provide the following information to your Active Directory Federation Services (ADFS) administrator:
```

Comment:
As far as I can tell, MS calls the product "Active Directory Federation Services", without the "Microsoft" or "Windows" words in it. Our dashboard also says only "ADFS".
Maybe "To connect your application to Microsoft's ADFS"?